catalog/imaging/imaging-postprocess-plugin-nitf/src/test/groovy/org/codice/alliance/plugin/nitf/NitfPostProcessPluginSpec.groovy

```diff
@@ -0,0 +1,406 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.plugin.nitf
+
+import ddf.catalog.data.Attribute
+import ddf.catalog.data.Metacard
+import ddf.catalog.data.types.Core
+import org.codice.ddf.catalog.async.data.api.internal.ProcessCreateItem
+import org.codice.ddf.catalog.async.data.api.internal.ProcessRequest
+import org.codice.ddf.catalog.async.data.api.internal.ProcessResource
+import org.codice.ddf.catalog.async.data.api.internal.ProcessUpdateItem
+import org.codice.ddf.catalog.async.data.impl.ProcessRequestImpl
+import org.codice.imaging.nitf.core.image.ImageSegment
+import org.codice.imaging.nitf.render.NitfRenderer
+import spock.lang.Ignore
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import java.awt.image.BufferedImage
+import java.util.function.Supplier
+
+class NitfPostProcessPluginSpec extends Specification {
+
+    private static final String GEO_NITF = "/i_3001a.ntf"
```

Comment:
❓Can we add a test with a NITF that has more than one segment to properly cover the `render()` method (L352)?

Reply:
❓Can we also have one that would exercise L474 - 483 in `renderToJpeg2k()`?

Reply:
It would be good to have some tests for these cases but probably needs another ticket since that's outside the scope of this one.

Reply:
I'm ok with creating a separate ticket, as long as we take care of it soon :-)